dashboard-parent/dashboard-core/src/main/java/module-info.java

```diff
@@ -1,11 +1,12 @@
 module org.wicketstuff.dashboard {
-	exports org.wicketstuff.dashboard;
-	exports org.wicketstuff.dashboard.web;
-	exports org.wicketstuff.dashboard.web.util;
+    exports org.wicketstuff.dashboard;
```

Comment:
@mcr-paulanand I'm afraid this is some sort of autoformatting? `tabs -> spaces` ?

I would leave indentation as-is :))

Reply:
Okay will do